mackup/utils.py

```diff
@@ -206,7 +209,7 @@ def get_dropbox_folder_location():
             data = f_hostdb.read().split()
     except IOError:
         error("Unable to find your Dropbox install =(")
-    dropbox_home = base64.b64decode(data[1])
+    dropbox_home = base64.b64decode(data[1]).decode('utf8')
```

Comment:
`base64.b64decode(data[1])` returns bytes under py3. `decode` here for getting a unicode string, no matter the environmet is  py2 or py3.